### ISSUE TITLE
Incompatibility with Monk's Enhanced Journal

### ISSUE THREAD

**Issue by KelSolaar** [mouse0270/lib-themer#38]
Hello,

I was unable to theme [Monk's Enhanced Journal](https://github.com/ironmonk88/monks-enhanced-journal). Problematic areas are highlighted in green here:

![image](https://user-images.githubusercontent.com/99779/211450463-878ce22b-da97-498e-9f52-a125f0bb9cf3.png)

I tested a few themes for Foundry and none of them seem to work with it properly.

**Comment by KelSolaar** [comment]
Oh and I just came across this as I posted and see that you chimed in already: https://github.com/ironmonk88/monks-enhanced-journal/issues/423#issuecomment-1299469075

**Comment by mouse0270** [comment]
Yeah most modules that use custom UI elements or very heavily modify the defaults of foundrys style wont work. When it comes to Monks Enhanced Journal, this is a big one, cause the UI is kinda a mess. I can't really make any promises as it will take some time and its not something I would like to maintain personally. 

I can spend some time to see if I can do something similar for the Fantasy RPG UI module, where I got it looking okay-ish, but I can't make promises to support this module directly as I really don't want to have to maintain it if the module changes. I will take a look though.

You can always use libThemer to make your own theme files and support them, you really just need to know how json and css work and drop the files in your user directory.

**Comment by KelSolaar** [comment]
> You can always use libThemer to make your own theme files and support them, you really just need to know how json and css work and drop the files in your user directory.

Thanks @mouse0270, this is exactly what I'm looking at right now!

**Comment by mouse0270** [comment]
> Thanks @mouse0270, this is exactly what I'm looking at right now!

If you have any questions you can always ask here or DM me on discord Mouse0270#0270 and I'll be glad to help.
